### PR TITLE
Fix unhappy path benchmark schema journey

### PR DESCRIPTION
**Pull request by VirajP1002**
### What is the context of this PR?
The unhappy benchmark journey when run against main was where the following error was reported on the set-min-max-block page: Expected a (200) but got a (404) back when getting page: /questionnaire/people/{person_2}/set-min-max-block/?resume=True').

To fix this, a similar journey was followed to recreate an updated requests files and when running the benchmark tests locally, the error message does not appear.

### How to review
Run the benchmark tests using `make run` ensuring the file in the command is changed from `test_checkbox.json` to `test_benchmark_business_unhappy_path.json`

### CODE REVIEW

**Comment by VirajP1002** [comment]
[Fix unhappy path benchmark schema journey](https://trello.com/c/SAGNqaey/5987-fix-unhappy-path-benchmark-schema-journey)

File: requests/test_benchmark_business_unhappy_path.json [hunk 1]
```diff
@@ -155,7 +148,7 @@
             "url": "/questionnaire/people/add-person/",
             "data": {
                 "first-name": "Erling",
-                "last-name": "Haland"
+                "last-name": "Haaland"
```
Comment (berroar) [inline requests/test_benchmark_business_unhappy_path.json:151]: 😆

**Review by berroar — APPROVED** [review]
Works, LGTM! 👍

**Review by petechd — APPROVED** [review]
Journey fixed, all good👍 We should make a new release after this merges?

File: requests/test_benchmark_business_unhappy_path.json (hunk 1 above)
Comment (VirajP1002) [reply requests/test_benchmark_business_unhappy_path.json:151]: Had to fix it! 😆

**Pull request merged**